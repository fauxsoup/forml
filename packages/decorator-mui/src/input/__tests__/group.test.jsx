import GroupComponent from "../group";
import React from "react";
import { render } from "@testing-library/react";

describe("renders", function() {
    let form;

    beforeEach(function() {
        form = { type: "text" };
    });

    test('consistently', function() {
        const { container } = render(
            <GroupComponent form={form} />
        )

        expect(container).toMatchSnapshot();
    })
});
