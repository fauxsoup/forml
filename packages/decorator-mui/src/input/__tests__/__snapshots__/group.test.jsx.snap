// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders consistently 1`] = `
<div>
  <div
    class="MuiFormControl-root css-1lq4zch-MuiFormControl-root"
  />
</div>
`;
